lib/nats/io/client.rb

```diff
@@ -1021,12 +1024,26 @@ def single_url_connect_used?
     end
 
     def send_command(command)
+      ensure_connected!
```

Comment:
It might be tricky, but I wonder if there is anyway to detect the fork without putting this check in the fast path?